Geometry2d/Point.hpp

```diff
@@ -40,6 +42,11 @@ class Point {
     Point(const QPoint& other) : Point(other.x(), other.y()) {}
 
     /**
+     * Implicit constructor for creating a Point from a float*
```

Comment:
It was necessary for k-d tree compatibility, but I recently added some functionality to the rrt's flann branch that does make this constructor unnecessary. I can go ahead and remove it we would prefer